domains/certificates/docker-test-env/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.8"
```

Comment:
We have a README-file in domains/certificates. Please add a section that we have this docker-compose and it is to be used for local development.